src/models/team.ts

```diff
@@ -3,8 +3,9 @@ import type { TournamentDeref } from '@/models/tournament';
 interface BaseTeam {
   id: number;
   name: string;
-  players: string[];
+  players: Record<string, string>[];
```

Comment:
You need to handle the non deref model where players is a list of ids. 
For example : `/v1/team/1/`